engines/mysql.py

```diff
@@ -64,9 +64,20 @@ def insert_data_from_file(self, filename):
 IGNORE """ + str(self.table.header_rows) + """ LINES
 (""" + columns + ")"
             try:
+                self.cursor.execute("""
+                  SET autocommit=0;
```

Comment:
so since we always have problems with bulk insert, in cases where the user has not set his database system for auto commit off, we can actually set it off here.
we can also set some other parameters off the may not let us do the bulk insert.